templates/wechatprogram/game.ejs

```diff
@@ -57,6 +58,7 @@ Page({
                     return firstScreen.setProgress(0.6).then(() => Promise.resolve(module));
                 }).then((cc) => {
                     require('./engine-adapter');
+                    require('src/chunks/bundle.js'); //bundle.js should reuqire after engine-adapter if use named register mode
```

Comment:
this is a hack, and it seems we don't know why we should reuqire after engine-adapter for now

Reply:
`src/chunks/bundle.js` maybe not exist after building project

Reply:
fixed. move to load-module file.